migration/migrator/data/course_tables.sql

```diff
@@ -2404,6 +2418,14 @@ ALTER TABLE ONLY public.peer_feedback
     ADD CONSTRAINT peer_feedback_pkey PRIMARY KEY (g_id, grader_id, user_id);
 
 
+--
+-- Name: peer_grading_panel peer_grading_panel_pkey; Type: CONSTRAINT; Schema: public; Owner: -
+--
+
+ALTER TABLE ONLY public.peer_grading_panel
```

Comment:
Did you manually edit `course_tables.sql`file to add constraint? I am reviewing the code, and it looks like your migration `20240508080451_add_peer_grading_panels_table.py` does not add any constraints but .sql file has them. You should not edit the .sql file directly, but rather use migrator dump to edit it. Here is the [documentation.](https://submitty.org/developer/development_instructions/migrations)

Reply:
@GO-viper7 Please let us know :)